tests/parsers_test.py

```diff
@@ -11,7 +11,7 @@
 SAMPLES_PATH = os.path.join(os.path.dirname(__file__), 'samples')
 SAMPLES = {}
 for extension in ['.spe', '.spc', '.cnf']:
-    filenames = glob.glob(os.path.join(SAMPLES_PATH, '*.*'))
+    filenames = glob.glob(os.path.join(SAMPLES_PATH + '*', '*.*'))
```

Comment:
@markbandstra was this not catching all files?

Reply:
I kept the original files in `samples` and put the converted ones in `samples_converted`. We can do this differently but I wanted the provenance to be clear.